Basic/Binding/Basic/Http/CS/Service/Service.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <AspNetCoreHostingModel>InProcess</AspNetCoreHostingModel>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore" Version="2.2.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Authentication" Version="2.2.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Authentication.Negotiate" Version="6.0.6" />
+    <PackageReference Include="CoreWCF.Http" Version="1.0.0" />
+  </ItemGroup>
```

Comment:
This should point to the latest version. Does 1.* work? If not, then use 1.2.* as that's what we just released.

Reply:
Using latest version 1.* works fine.